run_test.py

```diff
@@ -86,6 +86,8 @@ def main():
     else:
         numpy_min_version = '1.15'
 
+    numpy_newest_upper_version = '1.19'
```

Comment:
Lets lave this as 1.17.5 to avoid the decorators warning while we fix it in chainer

Reply:
Switching with CuPy's version might be preferred.
```
    if version.get_cupy_version() < (8,):
        numpy_min_version = '1.9'
        numpy_newest_upper_version = '1.18'
    else:
        numpy_min_version = '1.15'
        numpy_newest_upper_version = '1.19'
```

Reply:
I think that by just raising scikit it should be fine. 
Let me find a version where this does not happen.

Reply:
I think we should not expand the supported NumPy version for Chainer/CuPy v7 series anymore.

Reply:
Yes, that also works, latest working version is 1.17.5